tgui/.yarnrc.yml

```diff
@@ -1 +1 @@
-yarnPath: .yarn/releases/yarn-2.3.3.cjs
+yarnPath: .yarn/releases/yarn-2.4.0.cjs
```

Comment:
Revert changes to this file and `.yarn/releases/yarn-2.x.x.cjs`, we don't want this to conflict with CBT for no reason, and upgrading yarn is not necessary for it to function.

Reply:
Yup, handled that yesterday.